Dockerfile.yaml

```
@@ -0,0 +1,7 @@
+version: '3.9'
+
+services:
+  my-app:
+    build: .
+    ports: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)